roles/confluent.variables/defaults/main.yml

```diff
@@ -302,9 +311,10 @@ kafka_broker_rest_proxy_enabled: "{{confluent_server_enabled}}"
 
 #### Schema Registry Variables ####
 
-### Only use to customize Linux User Schema Registry Service runs with. User must exist on host.
+### Only use to customize Linux User Schema Registry Service runs with. Default user is cp-schema-registry
```

Comment:
Modify this variable to customize the Linux User that the Schema Registry Service runs with. Default user is cp-schema-registry.